### DISCUSSION TITLE
How do you change the layout of signature help?

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#1658] [General]
I want to know how I can change the colors, borders etc. when using signature help on a piece of code. Right now, the background color is almost (if not) identical to the background color of my theme (tokyonight) and it's really hard to see spot the signature pop-up.

**Comment by dimas-cyriaco** [comment]
This solved this problem to me:

```lua
return {
  "folke/noice.nvim",
  opts = {
    presets = {
      lsp_doc_border = true,
    },
  },
}

```

**Reply by mendesbarreto** [reply]
Thank you so much this is exactly what I needed =)
